src/components/Branches/BEYOIN/BeyoinMemberProfile.js

```diff
@@ -21,13 +21,13 @@ const BeyoinMemberProfile = (props) => {
     
       useEffect(() =>{
         
-        let member = befangYouthsInitiative.beyoin.find(member => member.id == memberId)
+        let member = befangYouthsInitiative.beyoin.find(member => member.id === memberId)
           setMember(member)
          }
         
          , [memberId])
           
-         if(member.img == "") return null;
+         if(member.img === "") return null;
```

Comment:
Please also check to ensure that this does not break the normal functioning of the component.
You can refactor this piece of code like this:
**if(!member.img) return null;**